www/SQLitePlugin.js

```diff
@@ -221,33 +225,46 @@
               
             };
         };
-      })(this);
-      openerrorcb = (function(_this) {
-        return function() {
+      openerrorcb = function() {
+          failed = true;
```

Comment:
Aren't we missing a `if (!failed)` check here, to avoid calling the error handler multiple times?